identifiers/country-us/state-ca-local_gov.csv

```diff
@@ -497,3 +497,64 @@ ocd-division/country:us/state:ca/place:visalia/council_district:2,Visalia CA cit
+ocd-division/country:us/state:ca/place:fullerton/council_district:5,Fullerton City Council Member District 5
+ocd-division/country:us/state:ca/place:garden_grove/council_district:1,Garden Grove City Council Member District 1
+ocd-division/country:us/state:ca/place:garden_grove/council_district:2,Garden Grove City Council Member District 2
+ocd-division/country:us/state:ca/place:garden_grove/council_district:3,Garden Grove City Council Member District 3
+ocd-division/country:us/state:ca/place:garden_grove/council_district:4,Garden Grove City Council Member District 4
+ocd-division/country:us/state:ca/place:garden_grove/council_district:5,Garden Grove City Council Member District 5
+ocd-division/country:us/state:ca/place:garden_grove/council_district:6,Garden Grove City Council Member District 6
+ocd-division/country:us/state:ca/place:oceanside/council_district:1,Oceanside City Council Member District 1
+ocd-division/country:us/state:ca/place:oceanside/council_district:2,Oceanside City Council Member District 2
+ocd-division/country:us/state:ca/place:oceanside/council_district:3,Oceanside City Council Member District 3
+ocd-division/country:us/state:ca/place:oceanside/council_district:4,Oceanside City Council Member District 4
+ocd-division/country:us/state:ca/place:orange/council_district:1,Orange City Council Member District 1
+ocd-division/country:us/state:ca/place:orange/council_district:2,Orange City Council Member District 2
+ocd-division/country:us/state:ca/place:orange/council_district:3,Orange City Council Member District 3
+ocd-division/country:us/state:ca/place:orange/council_district:5,Orange City Council Member District 5
+ocd-division/country:us/state:ca/place:oxnard/council_district:1,Oxnard City Council Member District 1
+ocd-division/country:us/state:ca/place:oxnard/council_district:2,Oxnard City Council Member District 2
+ocd-division/country:us/state:ca/place:oxnard/council_district:3,Oxnard City Council Member District 3
+ocd-division/country:us/state:ca/place:oxnard/council_district:4,Oxnard City Council Member District 4
+ocd-division/country:us/state:ca/place:oxnard/council_district:5,Oxnard City Council Member District 5
+ocd-division/country:us/state:ca/place:oxnard/council_district:6,Oxnard City Council Member District 6
+ocd-division/country:us/state:ca/place:rancho_cucamonga/council_district:1,Rancho Cucamonga City Council Member District 1
+ocd-division/country:us/state:ca/place:rancho_cucamonga/council_district:2,Rancho Cucamonga City Council Member District 2
+ocd-division/country:us/state:ca/place:rancho_cucamonga/council_district:3,Rancho Cucamonga City Council Member District 3
+ocd-division/country:us/state:ca/place:rancho_cucamonga/council_district:4,Rancho Cucamonga City Council Member District 4
+ocd-division/country:us/state:ca/place:sunnyvale/council_district:2,Sunnyvale City Council Member District 2
```

Comment:
Districts 1,3, and 5 are missing for Sunnyvale, CA

Reply:
added districts 1, 3, and 5, but looks like these districts won't be in use & filled until post-November 2022 elections [](https://sunnyvale.ca.gov/government/council/default.htm) - just want to make sure the timing is right to add these?

Reply:
IMO it's better to add them now since they will be districts for their respective elections and there's a risk that we'd forget to add them if we wait.